src/patternfly/base/_themes.scss

```diff
@@ -6,38 +6,38 @@
   }
 
   &.pf-m-transparent-100 {
-    background-color: rgba($pf-v5-color-black-1000, .42);
+    background-color: rgba($pf-v6-color-black-1000, .42);
```

Comment:
Do we even still need this file? Is it used for those deprecated components still?

Reply:
NOPE!